docs/reference/adventureworks/armTemplates/auxiliary/policies.json

```diff
@@ -18097,6 +18097,142 @@
             }
           },
           "name": "Deny-MachineLearning-ComputeCluster-Scale"
+        },
+        {
+          "properties": {
+            "displayName": "Deny-IotHub-MinimalTlsVersion",
+            "policyType": "Custom",
+            "mode": "Indexed",
+            "description": "Enforces minimal tls versions for iot hub.",
+            "metadata": {
+              "version": "1.0.0",
+              "category": "Internet of Things"
+            },
+            "parameters": {
+              "effect": {
+                "type": "String",
+                "metadata": {
+                  "displayName": "Effect",
+                  "description": "Enable or disable the execution of the policy"
+                },
+                "allowedValues": [
+                  "Audit",
+                  "Disabled",
+                  "Deny"
+                ],
+                "defaultValue": "Deny"
+              },
+              "tlsVersions": {
+                "type": "Array",
+                "metadata": {
+                  "displayName": "TLS Versions",
+                  "description": "Specifies the TLS versions that should be enforced"
+                },
+                "defaultValue": [
+                  "1.2"
+                ]
+              }
+            },
+            "policyRule": {
+              "if": {
+                "allOf": [
+                  {
+                    "field": "type",
+                    "equals": "Microsoft.Devices/IotHubs"
+                  },
+                  {
+                    "anyOf": [
+                      {
+                        "field": "Microsoft.Devices/IotHubs/minTlsVersion",
+                        "exists": false
+                      },
+                      {
+                        "field": "Microsoft.Devices/IotHubs/minTlsVersion",
+                        "notIn": "[[parameters('tlsVersions')]"
+                      }
+                    ]
+                  }
+                ]
+              },
+              "then": {
+                "effect": "[[parameters('effect')]"
+              }
+            }
+          },
+          "name": "Deny-IotHub-MinimalTlsVersion"
+        },
+        {
+          "properties": {
+            "displayName": "Deny-IotHub-Sku",
+            "policyType": "Custom",
+            "mode": "Indexed",
+            "description": "Enforces iot hub SKUs.",
+            "metadata": {
+              "version": "1.0.0",
+              "category": "Budget"
```

Comment:
Should this not be "Internet of Things" to keep these grouped together by the service?

Reply:
We had this discussion also on the Machine Learning PR. This is why I made the change. Let me know what the preferred option is.
@krnese FYI.